### PR TITLE
Make vm stop, start... more async

### LINKED ISSUE #120: Refacto some CMDs more async
as in this issue #99 and this PRs #101, #119 make the same refactoring for:

use asyncTasks() to make multiple query at the same time

- [x]  exo vm start
- [x]  exo vm stop
- [ ]  exo vm reboot
- [ ]  exo vm reset
- [ ]  exo vm resize
- [ ]  exo vm scale

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
Fix #120
Use asyncTasks() to execute start stop operation

- [x] start

- [x] stop

- [x] reboot

- [x] scale

- [x] reset

- [x] resize

### CODE REVIEW

File: cmd/vm_start.go [hunk 1]
```diff
@@ -23,24 +22,28 @@ var vmStartCmd = &cobra.Command{
...
+		tasks := make([]task, 0, len(args))
 		for _, v := range args {
-			if err := startVirtualMachine(v, rescueProfile); err != nil {
-				errs = append(errs, fmt.Errorf("could not start %q: %s", v, err))
+			vm, err := getVirtualMachineByNameOrID(v)
+			if err != nil {
+				return err
 			}
-		}
+			state := (string)(egoscale.VirtualMachineStopped)
```
Comment (mcorbin) [inline cmd/vm_start.go]: nitpick: I would rename this "stoppedState"

File: cmd/vm_stop.go [hunk 2]
```diff
@@ -23,30 +22,34 @@ var vmStopCmd = &cobra.Command{
...
 			}
 
-			if err := stopVirtualMachine(v); err != nil {
-				errs = append(errs, fmt.Errorf("could not stop %q: %s", v, err))
+			vm, err := getVirtualMachineByNameOrID(v)
+			if err != nil {
+				return err
 			}
-		}
+			state := (string)(egoscale.VirtualMachineRunning)
```
Comment (mcorbin) [inline cmd/vm_stop.go]: Here again, the variable could be renamed `runningState`.

File: cmd/vm_start.go (hunk 1 above)
Comment (pierre-emmanuelJ) [reply cmd/vm_start.go]: in this case the variable is only used one line after, when you read `egoscale.VirtualMachineStopped` you understand clearly.
https://talks.golang.org/2014/names.slide

File: cmd/vm_stop.go (hunk 2 above)
Comment (pierre-emmanuelJ) [reply cmd/vm_stop.go]: same as above

**Pull request merged**